hlink/scripts/lib/conf_validations.py

```diff
@@ -181,7 +184,10 @@ def check_comparison_features(config, columns_available):
                     raise ValueError(
                         f"Within [[comparison_features]] the 'column_name' {cname} is not available from a previous [[column_mappings]] or [[feature_selections]]: {c}"
                     )
+
         comps.append(alias)
+    if duplicates != []:
+        raise ValueError(f"Alias names are not unique: {duplicates}")
```

Comment:
This works nicely for checking duplicates. I think we should also check that no alias is already in `columns_available`, which are columns made available by the `column_mappings` and `feature_selections` sections earlier.

Reply:
Does the change I just made fix that?

Reply:
Yep, that looks good.

One small thing to think about is `alias` may be `None` in this check. We could move it after the check for `if alias is None` to remove that possibility. But I don't think it is an issue either way, so we can leave it.